### PR TITLE
Fix for - 23979 (Update to find code in program.cs file in dot net 6 rather then start up )

### PR DESCRIPTION
**Pull request by smandia**

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=24217) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=24217) before we can accept your contribution.<br/><br/>:x: smandia [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=24217)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetCore.Docs?pullRequest=24217) it.</sub>

File: aspnetcore/tutorials/first-web-api.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ For Visual Studio for Mac, see the .NET 5 version of this tutorial.
 * Enter *Web API* in the search box.
 * Select the **ASP.NET Core Web API** template and select **Next**.
 * In the **Configure your new project dialog**, name the project *TodoApi* and select **Next**.
-* In the **Additional information** dialog, confirm that **Framework** is **.NET 6.0 (Long-term support)**, and select **Create**.
+* In the **Additional information** dialog, confirm that **Framework** is **.NET 6.0 (Long-term support)**, checkbox **Use controllers(uncheck to use minimal APIs) is ticked** and select **Create**.
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/first-web-api.md:72]:
```suggestion
* In the **Additional information** dialog:
  * Confirm the **Framework** is **.NET 6.0 (Long-term support)**.
  * Confirm the checkbox for **Use controllers(uncheck to use minimal APIs)** is checked.
  * Select **Create**.
```

File: aspnetcore/tutorials/web-api-javascript.md [hunk 2]
```diff
@@ -35,9 +35,9 @@ The `fetch` function returns a [Promise](https://developer.mozilla.org/docs/Web/
...
+1. Configure the app to [serve static files](/dotnet/api/microsoft.aspnetcore.builder.staticfileextensions.usestaticfiles#Microsoft_AspNetCore_Builder_StaticFileExtensions_UseStaticFiles_Microsoft_AspNetCore_Builder_IApplicationBuilder_) and [enable default file mapping](/dotnet/api/microsoft.aspnetcore.builder.defaultfilesextensions.usedefaultfiles#Microsoft_AspNetCore_Builder_DefaultFilesExtensions_UseDefaultFiles_Microsoft_AspNetCore_Builder_IApplicationBuilder_). The following highlighted code is needed in the *Program.cs*:
 
-    [!code-csharp[](first-web-api/samples/3.0/TodoApi/StartupJavaScript.cs?highlight=8-9&name=snippet_configure)]
+    ![Program file](first-web-api/_static/EnableDefaultMappingAndServeStaticFiles.png)
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/web-api-javascript.md:40]: We never use images for snippets. The update to .NET 6 is tracked in #23979
Can you delete the `aspnetcore/tutorials/first-web-api/_static/EnableDefaultMappingAndServeStaticFiles.PNGaspnetcore/tutorials/first-web-api/_static/EnableDefaultMappingAndServeStaticFiles.PNG` file you added and revert this change. Accept my suggestion and I'll approve this PR.

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

File: aspnetcore/tutorials/web-api-javascript.md (hunk 2 above)
Comment (smandia) [reply aspnetcore/tutorials/web-api-javascript.md]: Reverted the files as mentioned.
1. I wish to take up issue tracked in #23979 . Was the usage of images for snippets only mistake or I need to add the whole solution in order to link to code in files.

Comment (Rick-Anderson) [reply aspnetcore/tutorials/web-api-javascript.md]: @smandia  that would be fantastic if you could take #23979 . Let me prepare the doc for you first, give me an hour. And yes, you use a new project. In this case, I think we copy the completed project from https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/tutorials/first-web-api/samples/6.0
But I'll copy that over for you.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**